Makefile

```diff
@@ -0,0 +1,9 @@
+# For Python bindings ===========================================================
```

Comment:
Very nice to have this. It adds support for make to the dependencies, but I suppose it could be seen as more of a convenience. (On windows you need to install make support)